insalan/payment/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path
+
+from . import views
+
+
+app_name = "payment"
+urlpatterns = [
+    # Flow URLs
+    path("pay/", views.PayView.as_view(), name="pay"),
+    path("notifications/", views.Notifications.as_view(), name="notifications"),
+
+    # REST URLs
+    path("product/", views.ProductList.as_view(), name="list-product"),
+    path("product/<int:pk>/", views.ProductDetails.as_view(), name="product-details"),
+    path("product/new/", views.CreateProduct.as_view(), name="new-product"),
+    path("transaction/", views.TransactionList.as_view(), name="transactions"),
+    path("transaction/<int:pk>", views.TransactionPerId.as_view(), name="transactions/id"),
+    # User View Pages
+    path("back/", views.BackView.as_view(), name="transaction/back"),
+    path("return/", views.ReturnView.as_view(), name="transaction/return"),
+    path("error/", views.ErrorView.as_view(), name="transaction/error"),
```

Comment:
Do those views still make sense? Logic will go in `notifications`, the callbacks will be in the frontend

Reply:
we still need to handle errors and back from HA

Reply:
I honestly don't really know what to do with those. What am I even gonna show? Shouldn't that be the front-end's job? If I can use notifications to know a payment happened, then I don't ever need the user to come back to a return route

Reply:
we could directly send those to the frontend

Reply:
Done in https://github.com/InsaLan/backend-insalan.fr/pull/67/commits/307f5ad1cd5c4fa8dfbfce9828db44f8d8b3e43b

(it says front because im a clumsy lil' fox 🦊)